platform/macos/macos.cmake

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.19)
-set(CMAKE_OSX_DEPLOYMENT_TARGET "10.13")
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.15")
```

Comment:
Some parts of std::filesystem are available only since 10.15 (which is pretty old, release date is October 7, 2019).